iotdb-core/datanode/src/main/java/org/apache/iotdb/db/queryengine/plan/planner/plan/node/load/LoadSingleTsFileNode.java

```diff
@@ -202,16 +202,16 @@ public String toString() {
         + '}';
   }
 
-  public void clean() {
+  public void clean(final boolean isLoadSuccess) {
```

Comment:
Since a false "isLoadSuccess" will just skip the whole function, why don't we just judge outside?

Reply:
> Since a false "isLoadSuccess" will just skip the whole function, why don't we just judge outside?

Both methods are acceptable. Two if statements need to be added to the external judgment. There are already many if statements in the calling place, so they are done in clean.